packages/edit-site/src/components/page-pages/index.js

```diff
@@ -206,6 +206,18 @@ export default function PagePages() {
 	const [ view, setView ] = useView( postType );
 	const history = useHistory();
 
+	const [ selection, setSelection ] = useState( [] );
+
+	const {
+		params: { postId },
+	} = useLocation();
+	const [ postIdToSelect, setPostIdToSelect ] = useState();
```

Comment:
Can we use postId as default value here to avoid initial re-rendering?

Reply:
Yes, good catch the code was updated 👍